src/GitHub.TeamFoundation.14/Services/TeamExplorerServices.cs

```diff
@@ -66,10 +67,8 @@ public void ShowHomePage()
 
         public void ShowPublishSection()
         {
-            var te = serviceProvider.TryGetService<ITeamExplorer>();
-            var page = te.NavigateToPage(new Guid(TeamExplorerPageIds.GitCommits), null);
-            var publish = page?.GetSection(new Guid(GitHubPublishSection.GitHubPublishSectionId)) as GitHubPublishSection;
-            publish?.Connect();
+            IGitActionsExt2 gitActionsExt = serviceProvider.TryGetService<IGitActionsExt2>();
+            gitActionsExt?.InitializeOrPushRepositoryToGitService();
```

Comment:
I only want these two lines to be executed in the 16.0 scenario since this type won't be available in 14.0 or 15.0.